template/README.rst

```diff
@@ -0,0 +1,40 @@
+<demo_name> Demo
+=============
+Short blurb on what this demo does.
+
+Usage
+-----
+A simple command on how to run the demo. For example,
+::
+  python <demo_name>.py
+
+Code Overview
+-------------
+Give a general overview of how the code works.
```

Comment:
Note that the first two sections just describe the content but the next ones describe user actions (give, draw). Not necessary but nice to use the same for for all (e.g., Short blurb, Simple description, General overview, Notable parts of)

Reply:
good point